tests/lib/rules/no-unused-prop-types.js

```diff
@@ -38,6 +38,22 @@ ruleTester.run('no-unused-prop-types', rule, {
   valid: [
     {
       code: [
+        'const Inner = (props) => <span>{props.innerOne} {props.innerTwo}</span>;',
+        'const Outer = (props) => {',
+        '  let team = props.names.map(() => (',
+        '      <Inner innerOne={props.one} innerTwo={props.two} />',
+        '    ));',
+        '  return <ul>{team}</ul>;',
+        '};',
+        'Outer.propTypes = {',
+        '  names: PropTypes.array,',
+        '  one: PropTypes.string,',
+        '  two: PropTypes.string',
+        '};'
+      ].join('\n'),
+      parser: 'babel-eslint'
```

Comment:
nothing about this test case should require babel-eslint; can it use the default parser instead?